### PR TITLE
Adds basic tests checking that components render properly when given dummy data

### PR DESCRIPTION
**Pull request by brockross**

### CODE REVIEW

File: client/src/components/tests/App.test.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import {mount, shallow} from 'enzyme';
 
-import App from './App';
+import App from '../App';
```
Comment (bkwon94) [inline client/src/components/tests/App.test.js:4]: Looks like you moved your test files to a test directory. Any reason for this? It seems more organized.

File: client/src/components/tests/Carousel.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
```
Comment (bkwon94) [inline client/src/components/tests/Carousel.test.js:2]: Is enzyme required for Jest?

**Review by bkwon94 — APPROVED** [review]
The code itself looks good, nothing I would change for now. Im just curious what enzyme does as I have come across that as well.

File: client/src/components/tests/App.test.js (hunk 1 above)
Comment (brockross) [reply client/src/components/tests/App.test.js:4]: Yeah I noticed you mentioned that in my last PR! I had been following a testing guide that recommended keeping test files in the same folder as the components they're testing, mostly for easier pathing. But it felt disorganized, and the deliverable for test files ask for a 'link to test folder in github', so I decided to switch to this.

**Pull request merged**